pkg/handler/router.go

```diff
@@ -13,7 +13,7 @@ func NewRouter(cfg *model.Config, services *service.Services) *gin.Engine {
 	handlers := loadHandlers(cfg, services)
 
 	api := router.Group("/api/v1")
-	// TODO: api.Use(tokenRequired)
+	// fixme: api.Use(tokenRequired)
```

Comment:
예전에 제가 빠뜨린 것이긴 한데, 아래와 같이 ()가 있어야 할 것 같고요. ㅎㅎ
```suggestion
	// fixme: api.Use(tokenRequired())
```
본문에 쓰신대로 토큰 기능만 구현만 하고, 적용은 하지 않도록 주석 처리 그대로 유지하셨군요.
go test만으로는 프론트엔드와 통합된 기능을 검증하기 어렵기 때문에, 일단은 이대로 두어야 하겠고요.

다만 이번 기회에 프론트엔드 측에 준비된 로직을 설명드립니다.

로그인할 때 token, userID, username을 로컬스토리지(브라우저)에 저장하고 axios(ajax 라이브러리) 헤더에도 세팅합니다. 반대로 로그아웃할 때는 초기화하게 되어 있고요.
https://github.com/kuoss/venti/blob/v0.2.0/web/src/stores/auth.ts#L22-L28

axiosPlugin(axios를 래핑하여 만든 vue plugin)에서는...
ajax 통신을 intercept하여 HTTP 401(권한 없음) 응답을 받는 경우에
token과 userID을 로컬스토리지에서 제거하게 되어 있고요.
https://github.com/kuoss/venti/blob/v0.2.0/web/src/plugins/axiosPlugin.ts#L47-L51

대략 이렇게 준비는 되어 있지만 전체적으로 잘 연결되어 있지는 않습니다.
이번 PR로 백엔드 쪽이 준비되었으니, 다음에 프론트엔드 쪽과 연계하여 구현해볼 수 있겠네요.

Reply:
안그래도 프론트 구현이 어떤지 궁금했는데 감사합니다. 다음에 연계 테스트를 해볼 수 있을 것 같네요!

Reply:
tokenRequired() 확인했습니다. resolve합니다.